compliance-report-os-is-vendor-supported/os-is-vendor-supported.json

```diff
@@ -16,13 +16,20 @@
     }
   },
   "conditions": {
-    "os-is-reported": {
-      "id": "os-is-reported",
+    "os-is-reported-and-not-unknown": {
+      "id": "os-is-reported-and-not-unknown",
       "name": "OS is reported",
-      "description": "All hosts must report their own OS",
+      "description": "All hosts must report their own OS and not be unknown",
```

Comment:
If a host reports "Unknown", did it really report its OS?

I'd consider keeping the same name / ID here, I don't see much value in renaming the condition (actually introducing a separate condition for those who have the previous one already).

Reply:
Well, not actually sending anything is quite different to me than unknown. And it seemed like updating the condition name to reflect the rules was most appropriate.

But it doesn't really matter too much to me. I can update the pr tomorrow.

Reply:
DONE.